Tapawingo_backend/Controllers/RoutepartsController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Tapawingo_backend.Dtos;
+using Tapawingo_backend.Models;
+using Tapawingo_backend.Services;
+
+namespace Tapawingo_backend.Controllers
+{
+    [ApiController]
+    public class RoutepartsController : Controller
+    {
+        private readonly RoutepartsService _routepartsService;
+
+        public RoutepartsController(RoutepartsService routepartsService) {
+            _routepartsService = routepartsService;
+        }
+
+        [HttpPost("routes/{routeId}/routeparts")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(RoutepartDto))]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<IActionResult> CreateRoutepart(int routeId, [FromForm] CreateRoutepartDto model)
+        {
+            var routepart = await _routepartsService.CreateRoutepart(model, routeId);
+            return routepart != null ?
+                new ObjectResult(routepart) { StatusCode = StatusCodes.Status201Created } :
+                NotFound(new { message = "Routepart not found" });
```

Comment:
Wat als ik empty strings meegeef in mijn model? Dan wordt dit volgensmij nog niet opgevangen

Reply:
op het moment dat je iets wat niet invuld en het is required in de Dto krijg je deze foutmelding:
![image](https://github.com/luuk-royal/tapawingohike_backend/assets/80511425/ec333ff8-fa5a-453f-889d-1bb3b6a012dd)